lib/src/browser/disposable_browser.dart

```diff
@@ -204,6 +204,11 @@ class Disposable implements disposable_common.Disposable {
       _disposable.manageDisposer(disposer);
 
   @override
+  disposable_common.Disposable getManagedDisposable(
```

Comment:
I would prefer to call this something like `manageAndReturnDisposable` or `manageDisposableV2`. We should also deprecate the existing `manageDisposable` since there is really no good reason to have both.

If breaking changes weren't so difficult to do I would say just update the existing method.